### PR TITLE
Duration fix

### PR DESCRIPTION
**Pull request by LuigiTrevisan**

### CODE REVIEW

**Review by JoaoVitorBranco — APPROVED** [review]

File: src/modules/get_all_actions_by_ra/app/get_all_actions_by_ra_viewmodel.py [hunk 1]
```diff
@@ -41,15 +42,13 @@ def to_dict(self):
...
-    def __init__(self, actions: List[Action], ra: str):
-        self.actions = actions
-        self.ra = ra
+    def __init__(self, associated_actions: List[AssociatedAction]):
+        self.associated_actions = associated_actions
     
     def to_dict(self):
         return {
-            "actions": [ActionViewModel(action, ra=self.ra).to_dict() for action in self.actions],
+            "actions": [ActionViewModel(action).to_dict() for action in self.associated_actions],
```
Comment (JoaoVitorBranco) [inline src/modules/get_all_actions_by_ra/app/get_all_actions_by_ra_viewmodel.py:52]: Coloca: 
{
actions: [(Sem RA)],
ra: str,
message: str
}

File: src/modules/get_all_actions_by_ra/app/get_all_actions_by_ra_viewmodel.py [hunk 2]
```diff
@@ -1,38 +1,39 @@
 import datetime
```
Comment (JoaoVitorBranco) [inline src/modules/get_all_actions_by_ra/app/get_all_actions_by_ra_viewmodel.py:1]: tira

**Review by JoaoVitorBranco — CHANGES_REQUESTED** [review]

File: tests/shared/infra/repositories/test_action_repository_mock.py [hunk 3]
```diff
@@ -35,6 +35,7 @@ def test_get_all_actions_by_ra(self):
         assert len(actions) == len(expected)
         assert actions == expected
         assert all([type(action) == AssociatedAction for action in actions])
+        assert all([action.member_ra for action in actions])
```
Comment (JoaoVitorBranco) [inline tests/shared/infra/repositories/test_action_repository_mock.py:38]: all([action.member_ra == ra_escolhido for action in actions])

**Review by JoaoVitorBranco — CHANGES_REQUESTED** [review]

File: tests/shared/infra/repositories/test_action_repository_mock.py (hunk 3 above)
Comment (LuigiTrevisan) [reply tests/shared/infra/repositories/test_action_repository_mock.py:38]: ok

**Review by JoaoVitorBranco — APPROVED** [review]

**Pull request merged**